source/documentation/breaking-changes/random-with-units.md.erb

```diff
@@ -0,0 +1,36 @@
+---
+title: "Breaking Change: Random With Units"
+introduction: >
+  Sass `random($limit)` will no longer ignore `$limit` units in a future
+  version. Passing a `$limit` with units will be deprecated to ensure the
+  compilation results don't change once the new behavior lands.
+---
+
+The function [`random($limit)`] has historically ignored units in `$limit` which
+returned a unitless value. For example `random(100px)` would drop "px" and
+return a value like `42`.
+
+A future implementation of Dart Sass will stop ignoring units for the `$limit`
+argument will return a random integer with the same units.
+
+[`random($limit)`]: ../modules/math#random
+
+<% example(autogen_css: false) do %>
+  // Future Sass, doesn't work yet!
+  @debug math.random(100px); // 42px
+  ===
+  // Future Sass, doesn't work yet!
+  @debug math.random(100px)  // 42px
+<% end %>
+
+## Transition Period
+
+<% impl_status dart: '1.54.5', libsass: false, ruby: false %>
+
+First, we'll emit deprecation warnings for the previous uses of `random()` where
+the `$limit` argument has units. We'll suggest a way to preserve the existing
+behavior and a way that emulates the future implementation.
```

Comment:
"We" is confusing here. Is "we" the documentation author? The Sass team? The Sass compiler?

I'd say "The Sass compiler" explicitly.